.github/workflows/cd.yml

```diff
@@ -194,6 +210,18 @@ jobs:
           allowUpdates: true
           removeArtifacts: true
 
+      - name: release v3 templates' RC version to github
+        if: ${{ (contains(steps.version-change.outputs.CHANGED, 'templates@') || contains(steps.version-change.outputs.CHANGED, '@microsoft/teamsfx')) && github.event_name == 'workflow_dispatch' && github.event.inputs.preid == 'rc' }}
+        uses: ncipollo/release-action@v1.10.0
+        with:
+          token: ${{ secrets.CD_PAT }}
+          prerelease: true
+          name: "templatesv3@0.0.0-rc"
+          tag: "templates@0.0.0-rc"
```

Comment:
Seems it conflicts to current rc template?

Reply:
Indeed, I will use a new tag here.